src/api-utils.js

```diff
@@ -0,0 +1,11 @@
+const fetchArticles = () => {
+  return fetch('https://liams-nc-news.onrender.com/api/articles').then((res) => {
+    return res.json();
+
+  }).then((res) => { 
+    return res.articles
+   })
+} 
+
+
+export default fetchArticles
```

Comment:
Your probably going to have other api calls in here. Consider a named export here: `export const fetchArticles = ...` instead of `export default` then you can easily add more. 

You'll need to destructure it when importing if you make this change 👍